paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/BancontactSpec.kt

```diff
@@ -2,27 +2,32 @@ package com.stripe.android.paymentsheet.specifications
 
 import androidx.compose.ui.graphics.Color
 import com.stripe.android.paymentsheet.R
+import com.stripe.android.paymentsheet.specifications.FormItemSpec.SaveForFutureUseSpec
 
 internal val bancontactParamKey: MutableMap<String, Any?> = mutableMapOf(
     "type" to "bancontact",
     "billing_details" to billingParams
 )
 
+internal val bancontactNameSection =
```

Comment:
are these used elsewhere? otherwise seems fine to inline them

Reply:
I can't inline them because they are needed in the saveForFutureUseSpec